bindings/java/src/org/sleuthkit/datamodel/TimelineManager.java

```diff
@@ -650,7 +680,7 @@ private long addEventDescription(long dataSourceObjId, long fileObjId, Long arti
 							.ifPresent(newEvents::add);
 				} catch (DuplicateException ex) {
 					duplicateExists = true;
-					logger.log(Level.SEVERE, getDuplicateExceptionMessage(artifact, "Attempt to make artifact event duplicate"), ex);
+					logger.log(Level.INFO, getDuplicateExceptionMessage(artifact, "Attempt to make artifact event duplicate"), ex);
```

Comment:
I think this exception will occur only on double-posting of the same event information, which should only happen if some ingest module erroneously double-posts the same artifact.  Shouldn't that be left as 'SEVERE'?

Reply:
You could make it a warning, but it's not hurting anything.  We are not losing information,   @rcordovano can decide.

Reply:
We should reserve INFO for things that do not involve errors. In this case, the double posting is an error, so INFO is not appropriate.

Logging a WARNING here is intuitive and would be the right thing to do except for one development process catch. Our regression tests log new SEVERE messages, but not new WARNINGs. So we need to log SEVERE to be sure the programming error for a double posting comes to our attention. This does seem to come up fairly often in log level discussions.

I think that we can consider changing out process. That would require a script change and new "gold." We should write a story about the process change and make this a SEVERE message until we can do the new story, which can reference this code.

Reply:
The only reason this is SEVERE is that its the only way to catch that some code is erroneously calling Blackboard.postArtifact multiple times with the same artifact? 

I will change it back to SEVERE.